crates/tracing-instrumentation/Cargo.toml

```diff
@@ -31,17 +31,21 @@ restate-workspace-hack = { workspace = true }
 restate-types = { workspace = true }
 
 arc-swap = { workspace = true }
-console-subscriber = { version = "0.4.1", features = [ "parking_lot", ], optional = true }
+console-subscriber = { version = "0.4.1", features = ["parking_lot"], optional = true }
 futures = { workspace = true }
+http = { workspace = true }
 metrics = { workspace = true, optional = true }
 metrics-exporter-prometheus = { workspace = true, optional = true }
 metrics-util = { workspace = true, optional = true }
 nu-ansi-term = "0.50.1"
 opentelemetry = { workspace = true }
-opentelemetry-contrib = { workspace = true, features = [ "jaeger_json_exporter", "rt-tokio", ] }
-opentelemetry-otlp = { workspace = true, features = ["tls", "tls-roots"] }
+opentelemetry-contrib = { workspace = true, features = ["jaeger_json_exporter", "rt-tokio"] }
+opentelemetry-http = { version = "0.30.0", features = ["reqwest"] }
+opentelemetry-otlp = { workspace = true, features = ["http-json", "http-proto", "reqwest-client", "tls", "tls-roots"] }
 opentelemetry-semantic-conventions = { workspace = true }
 opentelemetry_sdk = { workspace = true, features = ["rt-tokio"] }
+reqwest = { workspace = true }
+restate-serde-util = { workspace = true }
```

Comment:
nit: convention is to bundle restate internal deps together at the top of the block (before the line break)

Reply:
Will do. Can you suggest any additions to my `.taplo.toml` that would enforce this guideline?

```toml
[[rule]]
include = ["**/Cargo.toml"]
keys = ["dependencies"]

[rule.formatting]
column_width = 999
reorder_keys = true
reorder_arrays = true
```

Reply:
Single line per item and sorted. TIL about taplo :)